src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -240,12 +239,28 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
                             "start": datetime.now(tz=timezone.utc),
                             "end": datetime.now(tz=timezone.utc),
                         }
+
+                    latest_deploy = None
+                    if threshold.environment:
+                        # NOTE: if a threshold has no environment set, we monitor from start of the release creation
+                        # If a deploy does not exist for the thresholds environment, we monitor from start of release creation
```

Comment:
so the default is monitoring from release creation. if there is no environment for the threshold or no deploy for the threshold environment we use latest deploy?

Reply:
Correct yes